spark-jobs/src/test/scala/za/co/absa/enceladus/common/version/VersionExtSuite.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.common.version
+
+import org.scalatest.{FlatSpec, Matchers}
+import za.co.absa.commons.version.Version
+import za.co.absa.commons.version.Version._
+
+class VersionExtSuite extends FlatSpec with Matchers {
+
+  import VersionExt._
+
+  val expectedFinalVersion = Version.asSemVer("1.2.3")
+
+  "VersionExt" should "correctly derive final version for SemVers" in {
+    val toBeFinalized = Seq(
+      "1.2.3-alpha", "1.2.3-alpha.1", "1.2.3-0.3.7", "1.2.3-x.7.z.92",
+      "1.2.3-dev", "1.2.3-rc.1", "1.2.3-alpha.beta", "1.2.3-alpha.beta+222", "1.2.3"
```

Comment:
`1.0.0+20130313144700` this does not pass the test but is one of the examples of SemVer [spec](https://semver.org/#spec-item-10)

I have a feeling this is due to commons implementation, but I would like a second opinion.

Reply:
Filed AbsaOSS/commons#14
Thanks Sasa!

Reply:
Nice, thanks for catching this!

Reply:
If https://github.com/AbsaOSS/commons/pull/16 is merged and released soon, I would upgrade the dependency and include the test for the BigInt-based SemVer, but it is not essential for the core functionality here, I'd say.

Reply:
New Version used.

Reply:
The new version of utils (Version) used, version string of `1.0.0+20130313144700` is part of the test case now.